packages/mdc-select/package.json

```diff
@@ -25,6 +25,7 @@
     "@material/ripple": "^0.44.0",
     "@material/rtl": "^0.42.0",
     "@material/shape": "^0.43.0",
+    "@material/textfield": "^0.44.0",
```

Comment:
what is this for?

Reply:
I am also curious about this, as we have thus far avoided having text field or select rely on each other (otherwise we would've avoided duplicating some sub-components here)

Reply:
Woops, removed